.github/workflows/build.yml

```diff
@@ -41,9 +41,9 @@ jobs:
         set -eux
         python -m pip install .
 
-        pytest -vv -r ap --cov tvb-ext-bucket
+        pytest -vv -r ap --cov tvb_ext_bucket
```

Comment:
Could we keep the same name convention as in the other repos: `tvbextbuket` to be the Python package name ?